jni/drivers/adapter.h

```diff
@@ -58,14 +58,23 @@ namespace DriverAdapter {
 
 #if DRIVER == 0
 
-// OpenSL (note if mocking was requested - unit testing - mock_opensl_io.h has been included)
+// OpenSL
 
+#ifdef MOCK_ENGINE
+// mocking requested, e.g. unit test mode
+#include "../tests/helpers/mock_opensl_io.h"
+
```

Comment:
Now that we have an adapter for OpenSL and AAudio, we need it to understand there is a third layer for the unit test mode.